rbac/management/seeds.py

```diff
@@ -61,20 +56,10 @@ def run_seeds(seed_type, schema_list=None):
 
     try:
         with concurrent.futures.ThreadPoolExecutor(max_workers=MAX_SEED_THREADS) as executor:
-            if schema_list:
-                tenants = Tenant.objects.filter(schema_name__in=schema_list)
-            else:
-                tenants = Tenant.objects.all()
-            tenant_count = tenants.count()
-            for idx, tenant in enumerate(list(tenants)):
-                logger.info(
-                    f"Seeding {seed_type} changes for tenant {tenant.schema_name} [{idx + 1} of {tenant_count}]."
-                )
-                future = executor.submit(seed_functions[seed_type], tenant)
-                completed_log_message = (
-                    f"Finished seeding {seed_type} changes for tenant "
-                    f"{tenant.schema_name} [{idx + 1} of {tenant_count}]."
-                )
-                future.add_done_callback(partial(on_complete, completed_log_message, tenant))
+
+            logger.info(f"Seeding {seed_type} changes.")
+            future = executor.submit(seed_functions[seed_type])
```

Comment:
Here the function seed_roles, seed_group, seed_permissions still require tenant as input. Which will not run.

But this is included in my PR, should we merge this and get it fixed in my PR, or fix here? Both works for me. @wcmitchell @coderbydesign